Core/CMakeLists.txt

```diff
@@ -175,6 +177,8 @@ target_link_libraries(elastix_lib ${ELASTIX_TARGET_LINK_LIBRARIES})
 add_executable(transformix_exe
   Main/transformix.cxx
   Main/elastix.h
+  Main/elxMainExeUtilities.cxx
```

Comment:
It is somewhat unfortunate that Main/elxMainExeUtilities.cxx is built twice now: once for elastix_exe and once for transformix_exe.